### ISSUE TITLE
Assignee can't be blank

### ISSUE THREAD

**Issue by TFDuesing** [samchen2009/ekanban#23]
When creating a new issue with the ekanban plugin installed, the Assignee field becomes required, although it is not marked as such (with the red asterisk).

As a user, I'd like this field to not be required, for consistency with the standard Redmine installation.  Additionally, I can set this field to required by using a custom workflow, which seems like a more flexible approach to me.

**Comment by TFDuesing** [comment]
I feel it fitting to point out that GitHub allowed me to create this issue without setting an assignee.  :smirk:

**Comment by samchen2009** [comment]
Agreed that when an issue is 'new', the assignee can be blank (but I still think assigning to group is better if it's hard to determine the owner at that point), but in other status an assignee is a required to calculate WIP.  

This constraint was set at lib/ekanban/issue_patch.rb  line 57: validates_presence_of :assigned_to. 

I'll perfer to make it 'required' in view.

**Comment by dzacarias** [comment]
We found this problem while working with a project that doesn't use the plugin (it's managed in Scrum, with the Backlogs plugin)

Because of this, we could not try the plugin in our setup.

Our suggestion would be to only apply this validation in projects that have this plugin enabled.

**Comment by SteveDavis** [comment]
We have also hit this issue. Do you know if there's any chance of it being changed at all? Thanks!

**Comment by mileo** [comment]
@dzacarias I detect the same issue using, redmine blacklogs + ekanban

**Comment by ivansts** [comment]
I agree dzacarias, I haven´t used this plugin because affect other projects that use Backlogs plugin, in spite of being an interesting plugin.

Idem: redmine backlogs + ekanban

**Comment by dzacarias** [comment]
We've moved on to using Trello for Kanban at our company, so this issue's no longer "pressing" for us.

**Comment by ivansts** [comment]
Thanks Zacarias, my company is not allowed to use another tool different Redmine :(

If you know another way to configure  kanban on Redmine, you will be welcome.

**Comment by dzacarias** [comment]
I'm sorry, I haven't tried anything since moving on from this. Good luck!

**Comment by SteveDavis** [comment]
We came up with a slightly different approach – we’ve created a user of ‘Nobody Assigned’ and assign to that in order to get over the problem. Sure, it’s a bit of a fudge, but it works for us

Regards - Steve

**Comment by MauroAndreFlores** [comment]
That's too bad. I've liked this plugin. But because of this problem we are using the kanban of scrum2b
http://www.redmine.org/plugins/scrum2b.

**Comment by ivansts** [comment]
Thanks Mauro, I hadn´t taken into account. Using scrum2b could be another option.
